src/wabt.post.js

```diff
@@ -305,16 +315,16 @@ WasmModule.prototype.toText = function(options) {
   var writeModuleResult_addr =
       Module._wabt_write_text_module(this.module_addr, foldExprs, inlineExport);
 
-  var result = Module._wabt_write_module_result_get_result(
-      writeModuleResult_addr);
+  var result =
+      Module._wabt_write_module_result_get_result(writeModuleResult_addr);
```

Comment:
Unrelated change?   Maybe revert this is so?

Reply:
These were from clang-format, the linter complains otherwise.